index.html

```diff
@@ -0,0 +1,1783 @@
+                            </div>
+                            <div class="d-flex border-bottom mb-3">
+                                <div>
+                                    <img class="img-fluid" src="https://d.newsweek.com/en/full/1978865/josh-hammer.webp?w=63&h=63&f=6108b00453f3ccb5455bfd7a0e0bf3ba"
+                                        alt="">
+                                </div>
+                                <div class="ms-3">
+                                    <a class=" fw-bold text-dark h6 text-decoration-none" href="#">The Biden Regime Collapses the 'Public'-'Private' Distinction</a>
+                                    <p>BY JOSH HAMMER</p>
+                                </div>
+                            </div>
+        
+        
+                        </div>
+                        <div>
+                            <div class="position-relative mb-2">
+                                <img class="img-fluid" src="https://d.newsweek.com/en/full/1974985/josh-hammer-show.png?w=500&h=333&f=3c44e349ec0283d0da857b55f999f605"
+                                    alt="">
+                                <h6 class="text-primary bg-light p-2 text-center position-absolute bottom-0 mb-0" >PODCASTS</h6>
+                            </div>
+                            <a class="fw-bold text-dark h5 text-decoration-none" href="#">Rethinking Conservative Economics With Oren Cass</a>
+                            <p class="mt-2">Oren Cass, executive director of American Compass, joins Josh to discuss the Right's shifting
+                                approach to political
+                                economy and to wade into President Biden's disastrous student loan forgiveness initiative.</p>
+                        </div>
+                        <div class="bg-secondary p-4 mb-5">
+                            <div class="d-flex">
+                                <div>
+                                    <img class=" img-fluid" style="width: 1.5rem;" src="https://media-exp1.licdn.com/dms/image/C4E0BAQECoc6kRb5LJg/company-logo_200_200/0/1519856334420?e=2147483647&v=beta&t=c3-quZYRLVGisKweNzBbzgC4R2D8IuG84pZCm8IEtEU"
+                                    alt="">
+                                </div>
+                                <div>
+                                    <p class="fw-bold">GET THE BEST OF NEWSWEEK VIA EMAIL</p>
+                                </div>  
+                            </div>
+                            <div class="d-grid">
+                                <input type="email" name="email" id="email" placeholder="Email address">
+                            </div>
+                            <div class="d-grid mt-2">
+                                <button type="button" class="btn btn-danger">FREE SIGN UP</button>
+                            </div>
+                        </div>
+                        <div>
+                            <div class="position-relative mb-2">
```

Comment:
- [ ] No need to add position relative unless you are going to adjust the relative position of the element. You can check [this link](https://www.w3schools.com/css/css_positioning.asp) for more info.